TACC/Core-CMS
# feat: TUP-553 redesign news article byline

## Overview

Redesign the byline for news articles.

## Related

- [TUP-553](https://tacc-main.atlassian.net/browse/TUP-553)

## Changes

- **removed** custom translation for byline
- **removed** byline prefix from UI
- **tweaked** template comments for byline

## Testing

1. Open News article and News list.
2. Verify that the byline on both pages:
    1. does **not** have "by:" (nor "by")
    2. does have author name
    3. markup has "by" before author name
3. Verify that the byline on **list** page:
    1. is right-aligned to the right of date
4. Verify that the byline on **article** page:
    1. is left-aligned underneath date

## UI

| list page | article page |
| - | - |
| <img width="992" alt="list page" src="https://github.com/TACC/Core-CMS/assets/62723358/0afd1f25-d564-4280-b546-bb558fb0c6b0"> | <img width="992" alt="article page" src="https://github.com/TACC/Core-CMS/assets/62723358/63a9d694-7a3c-4ab0-a74a-47aae2e778f4"> |
